[2024-10-08T08:23:31.205-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-08T08:23:31.244-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T11:23:28.172666+00:00 [queued]>
[2024-10-08T08:23:31.261-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocess_predict.preprocess manual__2024-10-08T11:23:28.172666+00:00 [queued]>
[2024-10-08T08:23:31.484-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-08T08:23:31.856-0300] {taskinstance.py:2330} INFO - Executing <Task(_DockerDecoratedOperator): preprocess> on 2024-10-08 11:23:28.172666+00:00
[2024-10-08T08:23:31.867-0300] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=323) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-08T08:23:31.870-0300] {standard_task_runner.py:63} INFO - Started process 324 to run task
[2024-10-08T08:23:31.869-0300] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'preprocess_predict', 'preprocess', 'manual__2024-10-08T11:23:28.172666+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/Predict/preprocess_predict.py', '--cfg-path', '/tmp/tmpv60ko_81']
[2024-10-08T08:23:31.871-0300] {standard_task_runner.py:91} INFO - Job 114: Subtask preprocess
[2024-10-08T08:23:32.032-0300] {task_command.py:426} INFO - Running <TaskInstance: preprocess_predict.preprocess manual__2024-10-08T11:23:28.172666+00:00 [running]> on host 4a07270c49a4
[2024-10-08T08:23:32.464-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='preprocess_predict' AIRFLOW_CTX_TASK_ID='preprocess' AIRFLOW_CTX_EXECUTION_DATE='2024-10-08T11:23:28.172666+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-08T11:23:28.172666+00:00'
[2024-10-08T08:23:32.466-0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-08T08:23:32.716-0300] {docker.py:366} INFO - Starting docker container from image datapathpython:latest
[2024-10-08T08:23:32.725-0300] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-10-08T08:23:33.620-0300] {docker.py:436} INFO - + python3 -c 'import base64, os;x = base64.b64decode(os.environ["__PYTHON_SCRIPT"]);f = open("/tmp/script.py", "wb"); f.write(x);'
[2024-10-08T08:23:33.675-0300] {docker.py:436} INFO - + python3 -c 'import base64, os;x = base64.b64decode(os.environ["__PYTHON_INPUT"]);f = open("/tmp/script.in", "wb"); f.write(x);'
[2024-10-08T08:23:33.718-0300] {docker.py:436} INFO - + python3 /tmp/script.py /tmp/script.in /tmp/script.out
[2024-10-08T08:23:35.441-0300] {docker.py:436} INFO - Datos Preprocesados
[2024-10-08T08:23:38.740-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-08T08:23:39.090-0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=preprocess_predict, task_id=preprocess, run_id=manual__2024-10-08T11:23:28.172666+00:00, execution_date=20241008T112328, start_date=20241008T112331, end_date=20241008T112339
[2024-10-08T08:23:39.158-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-08T08:23:39.191-0300] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-08T08:23:39.196-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
